### PR TITLE
Adds minimum pop requirements for Power Sink and Sleeping Carp.

### PR DESCRIPTION
**Pull request by Moccha-Bee**
## About The Pull Request
Rebalances the power sink and sleeping carp by restricting it to a minimum pop recquirement.

## Why It's Good For The Game
Power sinks if used in low pops would spell doom to the station as there is not enough people to go find it and make the round umplayble and sleeping carp is too much for a low pop crew to deal with.

## Changelog
:cl:
balance: rebalanced the minimum pop recquirement for power sink and sleeping carp
/:cl:

### CODE REVIEW

**Comment by Lagomorphica** [comment]
no please moccha

sleeping carp is my main

i came here on my phone just to post this because i'm seething

you'd also have to add minimum pop for other strong as fuck items then no?

**Comment by zeskorion** [comment]
Lago other strong af items already have minpop


But yes fuck minpop reqs remove tbem all

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1278,13 +1279,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/card/emag
 	cost = 6
 
-/datum/uplink_item/device_tools/emag
-	name = "Blue Space Cryptographic Sequencer"
-	desc = "The cryptographic sequencer, electromagnetic card, or emag, is a small card that unlocks hidden functions,Comes with a remote use function \
-			in electronic devices, subverts intended functions, and easily breaks security mechanisms."
-	item = /obj/item/card/emag/bluespace
-	cost = 7
-
```
Comment (VictorVow) [inline code/modules/uplink/uplink_items.dm]: This change is out of scope

Comment (Moccha-Bee) [reply code/modules/uplink/uplink_items.dm]: left over from another pr removed it.

Comment (qwertyquerty) [reply code/modules/uplink/uplink_items.dm]: But you didn't

Comment (Moccha-Bee) [reply code/modules/uplink/uplink_items.dm]: I don't know what i did wrong but in the uplink file its not there. Maybe i am missing what you are talking about.

**Review by ike709 — CHANGES_REQUESTED** [review]
You shouldn't be touching any of the changelog files.

**Comment by Moccha-Bee** [comment]
> You shouldn't be touching any of the changelog files.

Well i dont remember changing the changelog files sorry for the inconvenice i will try fixing it.
